# Python headers are required to build Mercurial but weren't found in /opt/include/python3.11/Python.h
# ERROR Backend subprocess exited when trying to invoke get_requires_for_build_wheel

--- a/setup.py
+++ b/setup.py
@@ -660,11 +660,11 @@ class hgbuildpy(build_py):
             self.distribution.ext_modules = exts
         else:
             h = os.path.join(get_python_inc(), 'Python.h')
-            if not os.path.exists(h):
-                raise SystemExit(
-                    'Python headers are required to build '
-                    'Mercurial but weren\'t found in %s' % h
-                )
+#            if not os.path.exists(h):
+#                raise SystemExit(
+#                    'Python headers are required to build '
+#                    'Mercurial but weren\'t found in %s' % h
+#                )
 
     def run(self):
         basepath = os.path.join(self.build_lib, 'mercurial')
